content/01.abstract.md

```diff
@@ -1,3 +1,10 @@
 ## Abstract {.page_break_before}
 
-
+Microbial pangenomes reflect the metabolic diversity of groups of organisms.
+As *de novo* metagenome analysis techniques have matured and reference genome databases have expanded, metapangenome analysis has risen in popularity as a tool to organize the functional potential of organisms in relation to the environment from which those organisms were sampled.
+However, the reliance on assembly and binning or on reference databases often leaves substantial portions of metagenomes unanalyzed, thereby underestimating the functional potential of a community.
+To improve this problem, we present a method for metapangenomics that relies on protein k-mers (k~aa~-mers) and metagenome assembly graph queries.
```

Comment:
```suggestion
To address this challenge, we present a method for metapangenomics that relies on protein k-mers (k~aa~-mers) and metagenome assembly graph queries.
```

don't love my suggestion, but I don't think we want to improve a problem, either 😜